### PR TITLE
Adding mediaArtistEquals and mediaAlbumNameEquals options to mediaArtworkOverrides

### PR DESCRIPTION
**Pull request by gminco**
We use this card on a couple of panels around the house and we have little kids. Even though in Spotify you can filter explicit content, that doesn't apply to album covers, so having more broad options like artist and album will help hiding such covers.

### CODE REVIEW

**Comment by gminco** [comment]
@punxaphil Sorry about the 5 requests, I couldn't find the option to merge them to a single branch. Thanks!

File: src/sections/player.ts [hunk 1]
```diff
@@ -68,6 +68,8 @@ export class Player extends LitElement {
       let override = overrides.find(
         (value) =>
           (media_title && media_title === value.mediaTitleEquals) ||
+          (media_artist && media_artist === value.mediaArtistEquals) ||
```
Comment (punxaphil) [inline src/sections/player.ts:71]: looks like these are undefined? media_artist and media_album_name?
Did you actually try it?

**Comment by punxaphil** [comment]
Have you made sure this change works by testing it on your own setup?

File: src/sections/player.ts (hunk 1 above)
Comment (gminco) [reply src/sections/player.ts:71]: I did, but when I was trying to fix the mess of multiple pull requests I re-did the branch and I inadvertently forgot to copy one line.
I updated it.

**Comment by punxaphil** [comment]
Thanks again for contributing. Looks great!

**Comment by punxaphil** [comment]
https://github.com/punxaphil/custom-sonos-card/releases/tag/v7.10.0
Please try it out

**Comment by gminco** [comment]
I've just updated it from HACS, looking great! Thanks for being so responsive! And sorry about the multiple requests again.
I gotta say, this card is amazing, my family loves it! Thank you so much!

**Pull request merged**